spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -802,3 +802,120 @@ definitions:
+           type: u8
+           desc: >
+             Update interval between consecutive bounds. Encoded following
+             RTCM DF391 specification.
+       - sol_id:
+           type: u8
+           desc: SSR Solution ID. Encoded following RTCM DF415 specification.
+
+ - OrbitClockBound:
+     desc: Orbit and clock bound.
+     fields:
+       - sat_id:
+           type: u8
+           desc: >
+             Satellite ID. Encoded following either RTCM DF068 (GPS), DF252 (Galileo),
+             or DF488 (BDS) according to the constellation.
+       - orb_radial_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Mean Radial (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_along_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Mean Along-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_cross_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Mean Cross-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_radial_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Standard Deviation Radial (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_along_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Standard Deviation Along-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_cross_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Standard Deviation Cross-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - clock_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Clock Bound Mean (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - clock_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Clock Bound Standard Deviation (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+
+ - MSG_SSR_ORBIT_CLOCK_BOUNDS:
+     id: 0x05DE
+     short_desc: Combined Orbit and Clock Bound
+     public: False
+     fields:
+       - header:
+           type: BoundsHeader
+           desc: Header of a bounds message.
+       - ssr_iod:
+           type: u8
+           desc: IOD of the SSR bound. Encoded following RTCM DF413 specification.
```

Comment:
Are all of these values expected to use all 8 bits?  Would it make sense to pack these values into a larger type as a collection of bitfields?  This would allow use to more closely model the RTCM fields.

Reply:
Something similar to this: https://github.com/swift-nav/libsbp/blob/master/spec/yaml/swiftnav/sbp/system.yaml#L227 but instead of `flags` something like `metadata`

Reply:
Or probably more similar to this: https://github.com/swift-nav/libsbp/blob/master/spec/yaml/swiftnav/sbp/system.yaml#L200 -- since we won't be able to specify the values field in the yaml spec

Reply:
@silverjam do you know how this works for other already existing SBP messages which mimic RTCM fields?
For example SSR_CODE_BIASES here:
https://github.com/swift-nav/libsbp/blob/8a6c6216c105429d86c82976a5a88d5cabf5f190/spec/yaml/swiftnav/sbp/ssr.yaml#L293-L297
The `update_interval` is specified as a `u8`, but it also says it's encoded as DF391 which is only 4 bits.
I would think we still fill out all 8 bits, but the interpretation of it's value is then according to DF391?